mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/events/ElasticSearchListeningMessageSearchIndex.java

```diff
@@ -117,18 +117,19 @@ public void add(MailboxSession session, Mailbox mailbox, MailboxMessage message)
             indexer.indexMessage(indexIdFor(mailbox, message.getUid()), messageToElasticSearchJson.convertToJson(message, ImmutableList.of(session.getUser())));
         } catch (Exception e) {
             try {
-                LOGGER.warn("Indexing mailbox {}-{} of user {} on message {} without attachments ",
+                LOGGER.warn(String.format("Indexing mailbox %s-%s of user %s on message %s without attachments ",
```

Comment:
Ok you convaince me. But that's really suck from slf4j, they should have made a class to pack paceholder argument so we can pass them into only one argument!